src/chains/ethereum/ethereum/src/blockchain.ts

```diff
@@ -360,7 +360,7 @@ export default class Blockchain extends Emittery<BlockchainTypedEvents> {
         const mineAll = (maxTransactions: Capacity, onlyOneBlock = false) =>
           this.#isPaused()
             ? nullResolved
-            : this.mine(maxTransactions, null, onlyOneBlock);
+            : this.mine(maxTransactions, undefined, onlyOneBlock);
```

Comment:
Passing `undefined` means we have defined the value as `undefined`, and thus it is defined, with the value `undefined`. I think that is what `null` is for :-) Thoughts?

Also, I think passing `undefined` like this will not be allowed once we enable typescript's "strict" mode (https://github.com/trufflesuite/ganache/issues/3555).

Reply:
As discussed, this is probably up for a refinement of the `blockchain` interface. In the meantime, I've reordered the arguments to avoid needing to pass `undefined` or `null` explicitly.

(I also named the arguments within `evm_mine()` in api.ts because I found them confusing - open to dissenting opinions though!)